### ISSUE TITLE
Reemplazar avatares rotos

### ISSUE THREAD

**Issue by ricardoerl** [ricardoerl/svstamp#19]
Actualmente si la cuenta autora de algún tweet almacenado en svtamps cambia su avatar, la url del stamp de rompe y ya no se puede mostrar. La idea es encontrar una forma de tener un script o proceso que reemplace o actualice las urls de las imágenes en la base de datos.

**Comment by KevLehman** [comment]
I'll try to work in it during this week

**Comment by ralegria** [comment]
Ayer hice un mini-mini-research sobre esto... comparé la foto de los tweets del presidente en svstapms vs la foto actual del presidente en Twitter, y son diferentes. Lo que me lleva a pensar que talvez sea algo más el problema :thinking:

**Comment by ricardoerl** [comment]
@ralegria Si, justo la url ya no existe si la cuenta cambia su avatar. Con @KevLehman creo que se buscaria tener un job al final del día que actualice la base de imágenes.

**Comment by KevLehman** [comment]
Yup. El stamp guarda un enlace de la imagen, la cual parece que Twitter descarta cuando cambias la profile pic. 
La idea es dejar un job diario que traiga la imagen de la cuenta y actualice todas las Stamps de ese user.
